src/textual/widgets/_markdown.py

```diff
@@ -574,7 +577,7 @@ async def update(self, markdown: str) -> None:
         """
         output: list[MarkdownBlock] = []
         stack: list[MarkdownBlock] = []
-        parser = MarkdownIt("gfm-like")
+        parser = self._parser_factory()
```

Comment:
Does this still create a "gfm-like" parser?

Reply:
oh, no, it's totally wrong now :)

Reply:
It should be fixed now. thank you for the feedback!